web-app/src/components/DipMap.js

```diff
@@ -745,6 +745,110 @@ export default class DipMap extends React.Component {
 					this.debugCount("updateMap/hidProvince");
 				}
 			}
+
+
+			//Here we check each non-SC and non-Sea territory. If all surrounding SCs are of the same power and none is "Neutral", colour them that power.
+			//Get all nodes, disqualify Sea and SC, and per node collect the edges in an array.
```

Comment:
declare a variable here: `const nodes = this.state.variant.Properties.Graph.Nodes`
That way you can make 752, 753, and 757 a little bit more readable.